app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :post
+  belongs_to :user
+

```

Comment:
[OPTIONAL]
- Here you can add the `after_save :update_likes_counter` like you did for the posts so it can run immediately after the number of comments has updated. You can also make the function `private`.